libmscore/sym.cpp

```diff
@@ -6429,11 +6429,6 @@ void ScoreFont::draw(SymId id, QPainter* painter, const QSizeF& mag, const QPoin
 
       if (MScore::pdfPrinting) {
             if (font == 0) {
-                  QString s(_fontPath+_filename);
-                  if (-1 == QFontDatabase::addApplicationFont(s)) {
-                        qDebug("Mscore: fatal error: cannot load internal font <%s>", qPrintable(s));
-                        return;
-                        }
```

Comment:
Is this now just superfuous or would it harm to keep it?

Reply:
It seems to me superfluous because
1) It is loaded on init
2) If the user directory for SMuFL changes, there's a trigger that will re-apply this. 
3) I tested and fonts were working, also exported PDF and no problem

Reply:
OK ;-)

Reply:
Actually if it was kept, potentially the same crash I received would occur because it's not "bad code" but that MuseScore attempts, after calling addApplicationFont(s) to update TextCursor which can be "non-null" but not in  a totally valid state

Reply:
so even harmfull to leave it in